collector/cs.go

```diff
@@ -18,6 +18,9 @@ func init() {
 type CSCollector struct {
 	PhysicalMemoryBytes *prometheus.Desc
 	LogicalProcessors   *prometheus.Desc
+	Hostname            *prometheus.Desc
+	Domain              *prometheus.Desc
+	Fqdn                *prometheus.Desc
```

Comment:
These don't seem to be used?

Reply:
Removed, those were left overs of an earlier version with metrics for each value